public/docs/ts/latest/guide/architecture.jade

```diff
@@ -511,7 +510,7 @@ figure
   this `currency` pipe expression,
 <div style="margin-left:40px">
 code-example(language="javascript" linenumbers=".").
-  price | currency:'USD':true'
+  price | currency:'USD':true
```

Comment:
Unless I am missing something, the ending quote is invalid syntax, so I assumed it was a typo :smile: